src/main/steps/applicant/task-list/section-titles.ts

```diff
@@ -1,4 +1,8 @@
 export const applicant_en = {
-  applicantYourDetails: 'About you',
-  applicationDetails: 'Application detail',
+  aboutYou: 'About you',
+  yourApplication: 'Your application',
+  courtHearings: 'Your court hearings',
+  ordersFromCourt: 'Orders from the court',
+  yourDocuments: 'Your documents',
+  finalDecision: 'Final decision from the court',
```

Comment:
Please re-verify all the taskList items title as per the prototype again. I see few mismatch there.